src/main/webapp/app/modules/provider/record/single-record-view.tsx

```diff
@@ -232,6 +345,95 @@ class SingleRecordView extends React.Component<ISingleRecordViewProps, ISingleRe
+                      <div className="d-flex justify-content-between align-items-center">
+                        <FontAwesomeIcon icon="circle" className="orange" />
+                        &nbsp;
+                        <h5 className="mb-0">
+                          <b>
+                            {organization.services[currentServiceIdx].name
+                              ? organization.services[currentServiceIdx].name
+                              : translate('record.singleRecordView.noServiceName')}
+                          </b>
+                        </h5>
+                      </div>
+                      <div className="d-flex justify-content-between align-items-center">
+                        <div className="p-2 mr-1 arrow-navigation" onClick={() => this.prevService(servicesCount)}>
+                          <FontAwesomeIcon icon="angle-left" size="lg" />
+                        </div>
+                        <div className="d-flex justify-content-center mr-1">
+                          <div className="align-self-center">
+                            <b>{currentServiceIdx + 1}</b>
+                          </div>
+                          <div className="mx-2 align-self-center">
+                            <Progress value={((currentServiceIdx + 1) / servicesCount) * 100} />
+                          </div>
+                          <div className="align-self-center">
+                            <b>{servicesCount}</b>
+                          </div>
+                        </div>
+                        <div className="p-2 mr-5 arrow-navigation" onClick={() => this.nextService(servicesCount)}>
+                          <FontAwesomeIcon icon="angle-right" size="lg" />
+                        </div>
+                        <div className="p-2 arrow-navigation" onClick={this.closeServiceDetails}>
+                          <FontAwesomeIcon icon="times" size="lg" />
+                        </div>
+                      </div>
+                    </section>
+                    <section>
+                      <h6>
+                        <b>
+                          <Translate contentKey="record.singleRecordView.srvTypes" />
+                        </b>
+                        <span className="pill ml-2">
+                          {organization.services[currentServiceIdx].type
+                            ? organization.services[currentServiceIdx].type
+                            : translate('record.singleRecordView.untyped')}
+                        </span>
+                      </h6>
+                    </section>
+                    <section>
+                      <h6>
+                        <b>
+                          <Translate contentKey="record.singleRecordView.srvDescr" />
+                        </b>
+                      </h6>
+                      <span>{organization.services[currentServiceIdx].description}</span>
```

Comment:
Can you remove repeated organization.services[currentServiceIdx]?

Reply:
?